lib/ansible/modules/cloud/vmware/vmware_guest.py

```diff
@@ -2709,7 +2719,8 @@ def get_vm_events(self, vm, eventTypeIdList):
         eventManager = self.content.eventManager
         return eventManager.QueryEvent(filterSpec)
 
-    def wait_for_customization(self, vm, poll=10000, sleep=10):
+    def wait_for_customization(self, vm, cus_timeout=3600, sleep=10):
```

Comment:
I think parameter cus_timeout can be changed as customization_timeout. any specific reason it is made short?

Reply:
Oh, it's just used in that function not the parameter of the module, so I think short name is clear enough for coder. Thanks.

Reply:
`timeout` is also good option, if you want to make developer's life easy.

Reply:
Ok, use `timeout` to be simple and clear since it's inside the function. Thanks.